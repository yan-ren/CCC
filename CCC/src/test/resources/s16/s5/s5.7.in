15 100000000
001000000110001
