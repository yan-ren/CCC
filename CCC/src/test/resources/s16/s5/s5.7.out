010110010101101
